docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+
+Here is an example of how output metric will be written into VictoriaMetrics:
+`anomaly_score{for="node_cpu_rate", cpu="0", instance="node_exporter:9100", job="node-exporter", mode="idle"} 0.85`
+
+____________________________________________
+
+## VmAlert congiguration
+Here we provide an example of the config for VmAlert `vmalert_config.yml`.
+```
+groups:
+- name: AnomalyExample
+  rules:
+  - alert: HighAnomalyScore
+    expr: 'anomaly_score > 1.0'
+    labels:
+      severity: warning
+    annotations:
+      summary: Anomaly Score exceeded 1.0. `rate(node_cpu_seconds_total)` is showing abnormal behavior. 
+```
+
+In the query expression we need to put a condition on the generated anomaly scores. Usually if the anomaly score is between 0.0 and 1.0, the analyzed value is not abnormal. The more anomaly score exceeded 1 the more our model is sure that value is an anomaly.
+You can choose your threshold value that you consider reasonable based on the anomaly score metric, generated by VmAnomaly. One of the best ways is to estimate it visually, by plotting the `anomaly_score` metric, along with predicted "expected" range of `yhat_lower` and `yhat_upper`. Later in this tutorial we will show an example
+____________________________________________
+## Docker Compose configuration
+Now we are going to configure the `docker-compose.yml` file to run all needed services.
+Here are all services we are going to run:
+
+![Docker compose services](guide-vmanomaly-docker-compose.png "Docker compose services")
+
+* VictoriaMetrics - VictoriaMetrics Time Series Database
+* VmAgent - is an agent which helps you collect metrics from various sources, relabel and filter the collected metrics and store them in VictoriaMetrics or any other storage systems via Prometheus remote_write protocol.
+* Grafana - visualization tool.
+* Node exporter - Prometheus [Node Exporter](https://prometheus.io/docs/guides/node-exporter/) exposes a wide variety of hardware- and kernel-related metrics.
+* VmAlert - VictoriaMetrics Alerting service.
+* VmAnomaly - VictoriaMetrics Anomaly Detection service.
+
+### Grafana setup
+To anable VictoriaMetrics datasource as the default in Grafana we need to create a file `datasource.yml`
+```
+apiVersion: 1
+
+datasources:
+    - name: VictoriaMetrics
+      type: prometheus
+      access: proxy
+      url: http://victoriametrics:8428
+      isDefault: true
+
+```
+### Prometheus cofig
+Lets create `prometheus.yml` file for `vmagent` configuration.
+```
+global:
+  scrape_interval: 10s
+
+scrape_configs:
+  - job_name: 'vmagent'
+    static_configs:
+      - targets: ['vmagent:8429']
+  - job_name: 'vmalert'
+    static_configs:
+      - targets: ['vmalert:8880']
+  - job_name: 'victoriametrics'
+    static_configs:
+      - targets: ['victoriametrics:8428']
+  - job_name: 'node-exporter'
+    static_configs:
+      - targets: ['node_exporter:9100']
+  - job_name: 'vmanomaly'
+    static_configs:
+      - targets: [ 'vmanomaly:8500' ]
+```
```

Comment:
Better to add html div tags and code syntax highlighting for config examples - see
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/getting-started-with-vm-operator.md?plain=1#L23-L29 it will add copy button which is very useful for users.

Reply:
added div